Example/Standard Integration (Sources Only).xcodeproj/project.pbxproj

```diff
@@ -109,7 +109,7 @@
 		04823F791A6849200098400B /* Products */ = {
 			isa = PBXGroup;
 			children = (
-				04823F781A6849200098400B /* Standard Integration (Swift).app */,
+				04823F781A6849200098400B /* Standard Integration (Sources Only).app */,
```

Comment:
I checked out this branch but the example .xcodeproj are red & missing in the workspace - I still see the old "Custom Integration (ObjC)" and "Standard Integration (Swift)" files as well in Example/